paddle/fluid/framework/ir/mkldnn/conv_activation_mkldnn_fuse_pass.cc

```diff
@@ -26,6 +26,8 @@ using string::PrettyLogDetail;
 
 void ConvActivationMkldnnFusePass::ApplyImpl(Graph* graph) const {
   auto act_types = GetSupportedActivations();
+  act_types.erase(std::remove(act_types.begin(), act_types.end(), "sqrt"),
+                  act_types.end());
```

Comment:
Please remove OpCompat related to `sqrt`
- AddOpCompat(OpCompat("sqrt"))
- .EQ("sqrt", 0)